Hazelcast.Net/Hazelcast.IO.Serialization/SerializationConstants.cs

```diff
@@ -48,7 +48,10 @@ internal sealed class SerializationConstants
 
         public const int ConstantSerializersLength = 28;
 
-        public const int DefaultTypeSerializable = -110; //C# serializable
+        public const int DefaultTypeSerializable = -110; // C# serializable
+        public const int DefaultTypeJsonString = -130;
```

Comment:
On java side we used the name `JAVASCRIPT_JSON_SERIALIZATION_TYPE`. Can you match the same name?

Reply:
The name has been changed to `JavascriptJSONSerializationType`